scripts/imgtool/image.py

```diff
@@ -375,8 +375,9 @@ def create(self, key, public_key_format, enckey, dependencies=None,
                                     )
                     prot_tlv.add('DEPENDENCY', payload)
 
-            for tag, value in custom_tlvs.items():
-                prot_tlv.add(tag, value)
+            if custom_tlvs is not None:
```

Comment:
It looks OK, since you're at it, also fix the other `custom_tlvs` access on line 330 to check against `None`.

Reply:
Fixed.